### PR TITLE
Add K8s and sidecar version table for every feature

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

File: book/src/external-attacher.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
+--|--|--|--|--|--|--|--
+[external-attacher v1.1.1](https://github.com/kubernetes-csi/external-attacher/releases/tag/v1.1.1) | [release-1.1](https://github.com/kubernetes-csi/external-attacher/tree/release-1.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-attacher:v1.1.1 | v1.13 | - | v1.14
+[external-attacher v1.0.1](https://github.com/kubernetes-csi/external-attacher/releases/tag/v1.0.1) | [release-1.0](https://github.com/kubernetes-csi/external-attacher/tree/release-1.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-attacher:v1.0.1 | v1.13 | - | v1.13
+[external-attacher v0.4.2](https://github.com/kubernetes-csi/external-attacher/releases/tag/v0.4.2) | [release-0.4](https://github.com/kubernetes-csi/external-attacher/tree/release-0.4) | [v0.3.0](https://github.com/container-storage-interface/spec/releases/tag/v0.3.0) | [v0.3.0](https://github.com/container-storage-interface/spec/releases/tag/v0.3.0) | quay.io/k8scsi/csi-attacher:v0.4.2 | v1.10 | - | v1.10
```
Comment (pohly) [inline book/src/external-attacher.md]: Should v0.4.2 be removed? Is it still supported and/or needed?

File: book/src/sidecar-containers.md [hunk 2]
```diff
@@ -26,3 +26,34 @@ The Kubernetes development team maintains the following Kubernetes CSI Sidecar C
 * [node-driver-registrar](node-driver-registrar.md)
 * [cluster-driver-registrar](cluster-driver-registrar.md)
 * [livenessprobe](livenessprobe.md)
+
+
+## Versioning
```
Comment (saad-ali) [inline book/src/sidecar-containers.md]: The content here looks good. Maybe add subheadings so we can link to them directly

```
## Versioning

Every sidecar has a minimum, maximum and recommended Kubernetes version.

### Minimum Version
...

### Maximum Version
...

### Recommended Version
...
```

And then in the headers for the table can link directly to these subheadings here for those who are wondering what they mean

File: book/src/skip-attach.md [hunk 3]
```diff
@@ -2,8 +2,11 @@
 
 ## Status
 
-* Kubernetes 1.12: Alpha
-* Kubernetes 1.14: Beta
+Status | Min K8s Version | Max K8s Version | cluster-driver-registrar Version
+--|--|--
```
Comment (saad-ali) [inline book/src/skip-attach.md]: Need extra `|--`

File: book/src/external-attacher.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (msau42) [reply book/src/external-attacher.md]: Maybe I'll make "Recommend Version" a link to the sidecar page where I describe it

File: book/src/pod-info.md [hunk 4]
```diff
@@ -35,7 +38,9 @@ If the `podInfoOnMount` field is set to `true`, during mount, Kubelet will add t
 * `csi.storage.k8s.io/pod.namespace: {pod.Namespace}`
 * `csi.storage.k8s.io/pod.uid: {pod.UID}`
 
-The easiest way to use this feature is to deploy the [cluster-driver-registrar](cluster-driver-registrar.md) sidecar container. Once the flags to this container are configured correctly, it will automatically create a [CSIDriver Object](csi-driver-object.md) when it starts with the correct fields set.
+The CSIDriver object should be manually included in the driver manifests.
+
+Previously, the [cluster-driver-registrar](cluster-driver-registrar.md) sidecar container could be used to create the object. Once the flags to this container are configured correctly, it will automatically create a [CSIDriver Object](csi-driver-object.md) when it starts with the correct fields set.
```
[the remark this answers is not in the document]
Comment (msau42) [reply book/src/pod-info.md]: Luis has a pr here: https://github.com/kubernetes-csi/docs/pull/141

**Comment by jsafrane** [comment]
LGTM from my side, great update!

**Comment by bswartz** [comment]
Yes, this is helpful to me too. Thanks for updating the doc

**Pull request merged**